<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumPadLogInDialog</class>
 <widget class="QDialog" name="NumPadLogInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Senha</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>381</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="DuNumericKeyPushButton" name="setePushButton">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="quatroPushButton">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="umPushButton">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearPushButton">
          <property name="text">
           <string>CLEAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="DuNumericKeyPushButton" name="oitoPushButton">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="cincoPushButton">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="doisPushButton">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="zeroPushButton">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="DuNumericKeyPushButton" name="novePushButton">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="seisPushButton">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DuNumericKeyPushButton" name="tresPushButton">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enterPushButton">
          <property name="text">
           <string>ENTER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DuNumericKeyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">dunumerickeypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
